week-1/exercises/random.js

```diff
@@ -3,6 +3,11 @@ const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
+console.log(num);
+console.log(Math.floor(num));
+console.log(Math.floor(maximum - minimum + 1));
+console.log(Math.floor(Math.random() * (maximum - minimum + 1)));
+console.log(Math.floor(Math.random() * (maximum - minimum + 1)) + minimum);
```

Comment:
You haven't said what you think this function does

Reply:
I have added the explanation now. Please have a look!